### PR TITLE
Add tag images to the API

### LINKED ISSUE #560: Add label images to the API
There's currently no way to get label images (or packaging images, or PAO images) using the API

### PR DESCRIPTION
**Pull request by hangy**
Fixes #560

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -1241,12 +1241,19 @@ sub display_list_of_tags($$) {
...
+			my $tagentry = {
 				id => $tagid,
 				name => $display,
 				url => "http://$subdomain.$server_domain" . $product_link,
 				products => $products + 0, # + 0 to make the value numeric
 			};
+
+			if (defined $tags_images{$lc}{$tagtype}{get_fileid($icid)}) {
+				my $img = $tags_images{$lc}{$tagtype}{get_fileid($icid)};
+				$tagentry->{image} = "http://static.$server_domain/images/lang/$lc/$tagtype/$img";
```
Comment (stephanegigandet) [inline lib/ProductOpener/Display.pm]: should use the new sub for https/http now

**Comment by herau** [comment]
@stephanegigandet @hangy 

Don't find any label images in the production API.

For example: https://world.openfoodfacts.org/api/v0/product/3229820129488.json (which contains several bio labels).

Does this code is deployed in production ?

**Comment by hangy** [comment]
I added it to the `display_list_of_tags` API, see [preprod](http://world.preprod.openfoodfacts.org/labels.json).

**Comment by herau** [comment]
thanks @hangy 
When we hope find it in production ?
Can we will update the current documentation API for the new field please ?

**Comment by teolemon** [comment]
```{"url":"http://world.preprod.openfoodfacts.org/label/eu-organic","id":"en:eu-organic","image":"http://static.preprod.openfoodfacts.org/images/lang/en/labels/eu-organic.135x90.svg","products":8674,"name":"EU Organic"}```
Going to document it.
@stephanegigandet is handling production releases.

**Pull request merged**